project/npda/models/patient.py

```diff
@@ -148,8 +147,7 @@ class Meta:
             CAN_OPT_OUT_CHILD_FROM_INCLUSION_IN_AUDIT,
         ]
 
-    def clean(self):
-        super().clean()
+    def save(self, **kwargs):
         if not self.nhs_number and not self.unique_reference_number:
```

Comment:
Don't save at all if id fields are missing rather than just flagging it as an error